packages/server/src/routes/agencies.js

```diff
@@ -16,20 +17,36 @@ const {
 
 router.get('/', requireUser, async (req, res) => {
     const { user } = req.session;
-    let response;
-    if (user.role.name === 'admin') {
-        response = await getAgencies(req.session.selectedAgency);
-    } else {
-        response = await getAgency(req.session.selectedAgency);
+    const response = await getTenantAgencies(user.tenant_id);
+    res.json(response);
+});
+
+router.get('/impersonable', requireAdminUser, async (req, res) => {
```

Comment:
I'm confused about this endpoint:
1. It's not called anywhere? What is its intended use?
2. I thought based on the changes in access-helpers that we are now allowing impersonation of any agency in the tenant -- but this still uses `getAgencyTree` to return only subagencies of a user's agency.
3. What is the difference between `asAgency` and `organizationId` (aka `selectedAgency`)?

Reply:
So originally I had front-end changes to use this endpoint for the only purpose of selecting another agency to impersonate. I decided to move them to a later PR when taking out the super-tenant stuff and they were a little scrambled together. Should have mentioned that somewhere. I can remove these backend changes from this PR.